responses.py

```diff
@@ -312,6 +313,12 @@ def _url_matches(self, url, other, match_querystring=False):
         else:
             return False
 
+    def _post_params_match(self, post_params, request_body, content_type):
+        if post_params == None:
```

Comment:
To satisfy the linter, this should be changed to

```python
if post_params is None:
```

Reply:
nice catch, thank you! need more than a falsey comparison ({} would be valid to match on, for example)

Reply:
what's a good way to run the linter locally? is it pylint or similar?

Reply:
I think the linter used is [https://github.com/psf/black](url) if you want to run locally